src/test/java/com/amazon/opendistroforelasticsearch/knn/index/codec/KNNCodecTestCase.java

```diff
@@ -153,8 +153,8 @@ public void testMultiFieldsKnnIndex(Codec codec) throws Exception {
         IndexSearcher searcher = new IndexSearcher(reader);
         float score = searcher.search(new KNNQuery("test_vector", new float[] {1.0f, 0.0f, 0.0f}, 1, "dummy"), 10).scoreDocs[0].score;
         float score1 = searcher.search(new KNNQuery("my_vector", new float[] {1.0f, 2.0f}, 1, "dummy"), 10).scoreDocs[0].score;
-        assertEquals(score, 0.1667f, 0.01f);
-        assertEquals(score1, 0.0714f, 0.01f);
+        assertEquals(score, 1.0f/(1 + 25), 0.01f);
```

Comment:
assertEquals(expected, actual, ... )

Reply:
Good point. Im trying to fix these as I go. Will update.